docs/software/file_servers.md

```diff
@@ -120,6 +120,56 @@ The Samba server lets you share files on your DietPi system with ease based on t
     systemctl restart nmbd smbd
     ```
 
+=== "Use `wsdd` to view shares"
+
+    In case of problems with the Samba share not showing up in the Windows network view, the `wsdd` daemon (Web Service Dynamic Discovery host daemon) can be installed. This can be done with the following steps:
+
+    1. Install `wsdd`:
+       
+        ```sh
+        wget -O- https://pkg.ltec.ch/public/conf/ltec-ag.gpg.key | gpg --dearmour > /usr/share/keyrings/wsdd.gpg
+        source /etc/os-release
+        echo "deb [signed-by=/usr/share/keyrings/wsdd.gpg] https://pkg.ltec.ch/public/ ${UBUNTU_CODENAME:-${VERSION_CODENAME:-UNKNOWN}} main" > /etc/apt/sources.list.d/wsdd.list
+        apt update
+        apt install wsdd
+        ```
+
+    1. Configure `wsdd` (edit `/etc/systemd/system/wsdd.service`):
```

Comment:
Is this realy needed? I just installed wsdd and started the service. This already is showing up my DietPi system within Windows Explorer.

Reply:
I will check thits quickly in a VM.

Reply:
`systemctl start wsdd.service` should be enough

Reply:
`dietpi-services dietpi_controlled wsdd` means the service is disabled, while is should be enabled. Not sure whether it is OOTB, otherwise:
```sh
systemctl enable --now wsdd
```
Once it has been added to `dietpi-software` and then `dietpi-services`, it can be `dietpi_controlled` (=disabled).

Reply:
> `systemctl start wsdd.service` should be enough

This was not necessary to start the service, it ran directly after installing the wsdd package.
As far as I understood, adding the wsdd service to dietpi-services would enable DietPi tools (e.g. dietpi-backup) to stop and start the service from the script, isn*t it?

Reply:
personally, i don't see a need to stop the service.

Reply:
Ok.